integration/setup/client.go

```diff
@@ -39,9 +39,11 @@ func makeClient(ctx context.Context, uri string) (*mongo.Client, error) {
 		return nil, lazyerrors.Error(err)
 	}
 
-	if err = client.Ping(ctx, nil); err != nil {
-		return nil, lazyerrors.Error(err)
-	}
+	// The new PostgreSQL backend's tests can't survive this ping, as for every ping ANALYZE is called.
```

Comment:
That's an important moment in this PR. Unfortunately, with `ANALYZE` being called, tests stuck.

Reply:
That's not correct anymore